docs/ref/models/fields.txt

```diff
@@ -1359,8 +1359,8 @@ The possible values for :attr:`~ForeignKey.on_delete` are found in
 
     If ``limit_choices_to`` is or returns a :class:`Q object
     <django.db.models.Q>`, which is useful for :ref:`complex queries
-    <complex-lookups-with-q>`, then it will only have an effect on the choices
-    available in the admin when the field is not listed in
+    <complex-lookups-with-q>`.
+    It will not have and effect on choices in the admin if the field is listed
```

Comment:
If this is meant to be a new paragraph it needs a black line between it and the previous paragraph. Otherwise, flow it to the previous line.